backend/picamera_backend.py

```diff
@@ -61,13 +61,23 @@ def update_picamera_overlay(self, canvas: Canvas, layer: PiCameraOverlayLayer) -
         self.prev_overlays[layer] = overlay
 
     def on_base_canvas_updated(self, base_canvas: Canvas) -> None:
-        self.update_picamera_overlay(base_canvas, PiCameraOverlayLayer.base)
+        try:
+            self.update_picamera_overlay(base_canvas, PiCameraOverlayLayer.base)
+        except:
+            self.send_camera_error()
```

Comment:
Is there some weird python magic class/method where we can use the `self.send_camera_error()` method as the default exception handler? Just a bit annoying to wrap everything with a try except

Reply:
I knew there'd be some weird Python thing LOL

Reply:
Wow this is very intuitive, what the heck
![image](https://user-images.githubusercontent.com/58575113/88627728-22098880-d0f0-11ea-845c-971ac80dd653.png)
(This is just a random file I created btw, not the actual code lol)

I thought it would be some complicated thing, but this is as simple as it gets. Using this method, we'd have to keep using the `with` statement. Is this what you meant as the "default error handler"? @khanguslee

Reply:
I think I might've found a better way, but it's a little risky because it involves inheriting the Exceptions class and overwriting it. 

![image](https://user-images.githubusercontent.com/58575113/88629362-8299c500-d0f2-11ea-93dc-3e8fe7693a4d.png)

Reply:
What's the difference in the second method between that and just using the built-in `Exception` class? The first thing you suggested looks pretty neat and was what I was thinking of :)